MasterWindow.go

```diff
@@ -88,6 +92,7 @@ func (w *MasterWindow) setTheme() {
 	style := imgui.CurrentStyle()
 
 	imgui.PushStyleVarFloat(imgui.StyleVarWindowRounding, 2)
+	imgui.PushStyleVarFloat(imgui.StyleVarWindowBorderSize, 0)
```

Comment:
This line will set all window border size to zero. I think this may only affect when WindowFlagsFrameless is applied.

Reply:
Yes, this only take effect without the frame
I will remove this line from MasterWindow.go